src/cityreader/cityreader.py

```diff
@@ -20,8 +31,13 @@ def cityreader(cities=[]):
   # TODO Implement the functionality to read from the 'cities.csv' file
   # For each city record, create a new City instance and add it to the 
   # `cities` list
-    
-    return cities
+  with open('cities.csv') as csvfile:
+    readCSV = csv.reader(csvfile, delimiter=',')
+    next(readCSV, None)
+    name = ""
+    for row in readCSV:
+      cities.append(City(row[0], row[3], row[4]))
```

Comment:
Go ahead and change here just for future note the row[3] and [4 ] as float values as the test is expecting them in that format. Can also use eval(row[3]) and row 4 as well.